corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -88,7 +88,9 @@ class EntryInstances(PostProcessor):
 
     IGNORED_INSTANCES = {
         'jr://instance/remote',
+        'jr://instance/remote/remote',
         'jr://instance/search-input',
+        'jr://instance/search-input/search-input',
```

Comment:
Where are these new ones coming from?

Reply:
https://github.com/dimagi/commcare-hq/pull/32720/commits/e139d4e6380c8b2f71d40b428daca2fa1c7c7cfb commit added here since the catch for the `ValueError` was causing an error, from what I could see, that happens for certain references of legacy instances. But I don't the original context/intention of that catch